TreeCrownDetection/MainWindow.xaml.cs

```diff
@@ -176,19 +168,19 @@ private static unsafe Bitmap ToBitmap(IReadOnlyList<double> rawImage, int width,
             var sliderA = Application.Current.MainWindow.FindName("SliderA") as Slider;
             var sliderB = Application.Current.MainWindow.FindName("SliderB") as Slider;
 
-            int cutOff = 15;
-            int a = System.Convert.ToInt32(sliderA.Value);
-            int b = System.Convert.ToInt32(sliderB.Value);
+            var cutOff = 15;
+            var a = System.Convert.ToInt32(sliderA.Value);
+            var b = System.Convert.ToInt32(sliderB.Value);
```

Comment:
Rider has done it? I think, that better is to keep explicitly type of variable than use that auto type.

Reply:
It is explicite, you have type on the right side of the assignment.